sdks/python/apache_beam/io/textio_test.py

```diff
@@ -168,6 +168,28 @@ def test_read_single_file_larger_than_default_buffer(self):
     self._run_read_test(file_name, expected_data,
                         buffer_size=TextSource.DEFAULT_READ_BUFFER_SIZE)
 
+  def test_create_file_then_read(self):
+    with open('.txt', 'w') as file:
+      file.write('Hello\n')
+
+    p = beam.Pipeline()
+    output1 = p | "Read1" >> ReadFromText("./.txt")
+
+    with open('test_file.txt', 'w') as file:
+      file.write('Hello\n')
+
```

Comment:
Nit: following convention in this file, don't have newlines between each individual statement.